runtime/src/main/java/org/corfudb/runtime/BootstrapUtil.java

```diff
@@ -107,19 +107,18 @@ public static void bootstrap(@NonNull Layout layout,
                                  @NonNull Duration retryTimeout) {
         for (String server : layout.getAllServers()) {
             int retry = retries;
+
+            IClientRouter router = new NettyClientRouter(NodeLocator.parseString(server),
+                    corfuRuntimeParameters);
+            router.addClient(new LayoutHandler())
+                    .addClient(new ManagementHandler())
+                    .addClient(new BaseHandler());
+
             while (retry-- > 0) {
                 try {
                     log.info("Attempting to bootstrap node:{} with layout:{}", server, layout);
-                    IClientRouter router = new NettyClientRouter(NodeLocator.parseString(server),
```

Comment:
It was previously recreating because the creating router method used to throw NetworkException.  Now it doesn't so we don't need to recreate router every retry.